apps/search/views.py

```diff
@@ -165,16 +164,14 @@ def queryset(self):
                 pass
             qs = None
             if pk:
-                qs = Addon.objects.filter(id=int(q), disabled_by_user=False)
+                qs = Addon.objects.reviewed().filter(id=int(q))
             elif len(q) > 2:
                 # Oh, how I wish I could elastically exclude terms.
                 # (You can now, but I forgot why I was complaining to
                 # begin with.)
```

Comment:
Heh. I don't know. I was wondering that too, and considered removing it, but I wasn't quite sure what it meant.

But since you also seem not to know what it means, I guess I'll remove it.